assets/src/scripts/components/Layout/Sidebar.tsx

```diff
@@ -20,19 +20,22 @@ export default function Sidebar({
 }) {
   return (
     <Col className="builder__sidebar" md="3">
-      {!isEmptyCodelist && <Summary counts={counts} draftURL={draftURL} />}
-
-      {searches.length > 0 && (
-        <Searches
-          draftURL={draftURL}
-          isEditable={isEditable}
-          searches={searches}
-        />
+      {isEmptyCodelist ? (
```

Comment:
The logic inversion here took me a while to parse!

Reply:
I want to move a lot of this logic inside the components, but will do that when the components are cleaned up and refactored.